src/gaUtility.cpp

```diff
@@ -0,0 +1,19 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+//' Compute gaUtility (Goal angle utility)
+//'
+//' @param bGA integer
+//' @param aGA integer
+//' @param GA numeric vector
+//' @returns a numeric vector of length equal to GA's length 
+//' @export
+// [[Rcpp::export]]
+
+NumericVector gaUtility_rcpp(int bGA, int aGA, NumericVector GA) {
```

Comment:
Here, you declared `bGA` and `aGA` to be integers. In the examples, this might be true, but in general I think they can (and should) be doubles. In the other utility functions the 'a' and 'b' parameters are doubles as well, and I don't see a reason why they shouldn't be here as well. @n400peanuts what do you think?

Reply:
You're right! I have tried to see whether changing the type here made any difference and forgot to revert it back. I checked and the default type in R seems to be the **double** -- so it is smarter to assign it this way.

REF: https://www.geeksforgeeks.org/r-data-types/#:~:text=Numeric%20Datatype&text=It%20is%20the%20default%20data,will%20be%20of%20numeric%20type.